### PR TITLE
ltopers 2023-08-23

### PR DESCRIPTION
**Pull request by retokromer**

### CODE REVIEW

File: ltopers.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Ltopers < Formula
   desc "Scripts for doing stuff with LTFS"
   homepage "https://github.com/amiaopensource/ltopers"
-  url "https://github.com/amiaopensource/ltopers/archive/v2020-01-27.tar.gz"
-  version "2020-01-27"
-  sha256 "7d3ba39bab196e352070ce5a4457113bf8682cdbc54469bc16f8f498313781bb"
+  url "https://github.com/amiaopensource/ltopers/archive/refs/tags/v20230823.tar.gz"
```
Comment (privatezero) [inline ltopers.rb:4]: Is the change to `/archive/refs/tags/` from just `/archive/` in the HTML more robust?

**Review by privatezero — APPROVED** [review]
LGTM!

File: ltopers.rb (hunk 1 above)
Comment (retokromer) [reply ltopers.rb:4]: Thanks! No, I don’t think it’s more robust. I’ll change this.

**Comment by retokromer** [comment]
I have seen that Dave is currently updating these scripts. Therefore I’ll wait a few more days and see if another release is finalised. The current release is three and a half year old anyway …

**Comment by retokromer** [comment]
As to my knowledge the development is currently paused, I merge this pre-release. Another release could follow soonish.

**Pull request merged**